app/templates/macros/toolkit_forms.html

```diff
@@ -127,6 +127,24 @@
   {% endwith %}
 {%- endmacro %}
 
+{% macro date(question_content, data, errors) -%}
+  {%
+    with
+    name=question_content.id,
+    question=question_content.question,
+    question_advice=question_content.question_advice,
+    hint=(question_content.hint or ''),
```

Comment:
All the other macros in this file simply use `question_content.hint` rather than `(question_content.hint or '')`?